frontend/middleware_validatesubscription_test.go

```diff
@@ -16,13 +16,29 @@ import (
 	"github.com/google/go-cmp/cmp"
 
 	"github.com/Azure/ARO-HCP/internal/api/arm"
+	"github.com/Azure/ARO-HCP/internal/metrics"
 )
 
+type MockEmitter struct{}
```

Comment:
should this get extracted to a test/util package for importing in other test files using the `emitter`?